kpi/views/__init__.py

```diff
@@ -99,6 +99,7 @@ def one_time_login(request):
         expiry__lt=datetime.datetime.now()).delete()
     with transaction.atomic():
         try:
+            # FIXME: select_for_update!!!
```

Comment:
i thought i got rid of this entire authentication mechanism, or at least opened an issue about getting rid of it :weary:

Reply:
well, at least https://github.com/kobotoolbox/reports/pull/138 got done